src/components/multi-action-button/multi-action-button.js

```diff
@@ -95,6 +74,46 @@ class MultiActionButton extends SplitButton {
       'data-role': this.props['data-role']
     };
   }
+
+  render() {
+    return (
+      <StyledMultiActionButton buttonType={ this.props.buttonType || this.props.as }>
+        { super.render() }
+      </StyledMultiActionButton>
+    );
+  }
 }
 
+MultiActionButton.propTypes = {
+  /**
+   * Customizes the appearance, can be set to 'primary', 'secondary' or 'transparent'.
```

Comment:
I think we can remove not necessary spaces and keep comments in one line
`/* Customizes the appearance, can be set to 'primary', 'secondary' or 'transparent'. */`

Reply:
multi action button will be rewritten in a separate task, so I think we could leave it as it is now